packages/config/src/chains.ts

```diff
@@ -1257,7 +1255,7 @@ export const shiden = new Parachain({
   name: 'Shiden',
   parachainId: 2007,
   ss58Format: 5,
-  ws: 'wss://rpc.shiden.astar.network',
+  ws: 'wss://shiden-rpc.dwellir.com',
```

Comment:
I'm changing this because the other one was unstable and sometimes took a long time to load